src/lib/Preview.js

```diff
@@ -502,6 +507,11 @@ class Preview extends EventEmitter {
             this.retryCount = 0;
         }
 
+        if (this.previewOptions.file) {
+            this.loadPreviewWithTokens({});
+            return;
+        }
```

Comment:
So, when we pass well-formed file object, we need to call `this.loadPreviewWithTokens` but we don't pass the token and in lines below, we call `this.loadPreviewWithTokens` after executing `getTokens`, so as a proposal with according to your pull request https://github.com/box/box-content-preview/pull/328/, can I use guard here: 
```
if (typeof fileIdOrFile === 'object') {
     this.loadPreviewWithTokens({});
     return;
}
```
?